react-hash-routing/client/pages/product-details.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { toDollars } from '../lib';
+
+const styles = {
+  image: {
+    width: '100%',
+    height: '350px',
+    objectFit: 'contain'
+  },
+  longDescription: {
+    whiteSpace: 'pre-wrap'
+  }
+};
+
+export default class ProductDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      product: null
+    };
+  }
+
+  componentDidMount() {
+    fetch(`/api/products/${this.props.productId}`)
+      .then(res => res.json())
+      .then(product => this.setState({ product }));
+  }
+
+  render() {
+    if (!this.state.product) return null;
+    const {
+      name, imageUrl, price, shortDescription, longDescription
+    } = this.state.product;
+    return (
+      <div className="container">
+        <div className="card shadow-sm">
+          <div className="card-body">
+            <div className="row">
+              <div className="col">
+                {/* this anchor should go back to the catalog at '#' */}
+                <a href="" className="btn text-secondary">
```

Comment:
Missed an anchor tag

Reply:
Can't believe I missed that one.  Thanks Tim!